server/src/main/scala/controllers/package.scala

```diff
@@ -47,6 +47,17 @@ package object controllers {
     }
   }
 
+  def decodeAWSToken(token: String): (UUID, UUID) = {
```

Comment:
This is not an `AWS` token, I'd rename it to `decodeUserToken`, let's return `case class UserToken(userId: UUID, token: UUID)`